app/src/main/java/com/example/footballmanager/ui/FootballManagerApp.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.footballmanager.ui
+
+import android.annotation.SuppressLint
+import android.os.Build
+import android.widget.Toast
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.material3.TopAppBarScrollBehavior
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.nestedscroll.nestedScroll
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.example.footballmanager.screens.result_screens.success_components.Header
+import com.example.footballmanager.screens.HomeScreen
+import com.example.footballmanager.screens.view_models.HomeViewModel
+import java.time.LocalDate
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun FootballManagerApp() {
+    val scrollBehavior = TopAppBarDefaults.enterAlwaysScrollBehavior()
+    val homeViewModel: HomeViewModel = viewModel()
+
+    LaunchedEffect(key1 = true) {
+        //Toast.makeText(ctx, "Invoke retrieving data here", Toast.LENGTH_SHORT).show()
```

Comment:
remember to remove commented out code at the end of development (if it's not needed anymore). You have git history anyway if you would like to bring something back